wca-regulations.md

```diff
@@ -241,7 +241,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 9b5a) Full round formats for this event: "Best of X" (where X is 1, 2, or 3).
         - 9b5b) Cutoff formats for this event: "Best of X" (where X is 1 or 2) cutoff phase for "Best of Y" (where Y is 2 or 3, and Y > X).
 - 9f) The results of a round are measured as follows:
-    - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated to the nearest hundredth of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.
+    - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated to the hundredth of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.
```

Comment:
I'd go with this, as "the hundredth of a second" is 1/100, and "hundredths of a second" is a bit less overly specific.
```suggestion
    - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated down to hundredths of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.
```

Reply:
Thanks for taking a look!  I agree that just dropping the "nearest" left the phrasing a little clunky.  I appreciate your having taken a second run at the specific wording.

I like your use of the word "down" to help plant the idea that you'd not end up at 1.24 from 1.239 for example.

If you're completely happy with the phrasing here, so am I.